tests/unit/converters/test_ome_zarr.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+import numpy as np
+import pytest
+import zarr
+
+from tiledbimg.converters.ome_zarr import OMEZarrReader
+
+
+@pytest.fixture
+def mocker_tiff(mocker):
+    mocker.patch("zarr.open")
+
+
+class TestOMEZarrReader:
+    def test_ome_zarr_level_count(self, tmp_path):
```

Comment:
This test and the next one don't use the mocked `zarr.open`

Reply:
Yes I will add although it does not make a change since `zarr.open` accepts empty path as input, but for completeness I will add it